impl/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>com.okta.commons</groupId>
             <artifactId>okta-http-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
```

Comment:
Remove dependency on okhttp, some clients use `okhttp` and some use `httpclient` we have an abstraction in place to deal with this already ;)

Reply:
done